cli/bpmetadata/tfconfig_test.go

```diff
@@ -68,22 +81,46 @@ func TestTFInterfaces(t *testing.T) {
 	require.NoError(t, err)
 	for _, tt := range varTests {
 		t.Run(tt.name, func(t *testing.T) {
-			assert.Contains(t, got.Variables, BlueprintVariable{
-				Name:         tt.varName,
-				Description:  tt.wantDescription,
-				DefaultValue: tt.wantDefault,
-				Required:     tt.wantRequired,
-				VarType:      tt.wantVarType,
-			})
+			i := slices.IndexFunc(got.Variables, func(v *BlueprintVariable) bool { return v.Name == tt.varName })
+			if got.Variables[i].Name != tt.varName {
+				t.Errorf("getBlueprintInterfaces() - Variable.Name = %v, want %v", got.Variables[i].Name, tt.varName)
+				return
+			}
```

Comment:
Couldn't we just assert.Equal() each of these assertions?

Reply:
I like checking each attribute since it's easier to tell where exactly the failures occur.

Reply:
Resolving since we chatted offline